CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
+
+- New [i18n](https://github.com/bem/bem-core/tree/v2/common.blocks/i18n) block was introduced, providing support for internationalization ([#1074](https://github.com/bem/bem-core/issues/1074)).
+- Now jQuery is included via `https` by default ([#1202](https://github.com/bem/bem-core/issues/1202)).
+- Dependency on `bemhtml-compat` was dropped ([#1186](https://github.com/bem/bem-core/issues/1186)).
```

Comment:
There's no need in it in `bem-core` for users of `ENB` anyways.

Users of `bem-tools` who will update to latest `bem-core` version (if any) just need to run `npm i bemhtml-compat` to fix it on their projects. So the change looks quite harmless.